Jenkinsfile

```diff
@@ -748,7 +770,7 @@ def transform(String original, String transform) {
     """
   } else {
     bat """
-      powershell \"\$xml = Resolve-Path ${original};\$output = Join-Path (\$pwd) temp.xml;\$xslt = New-Object System.Xml.Xsl.XslCompiledTransform;\$xslt.Load(\"${transform}\");\$xslt.Transform(\$xml, \$output);\"
+      powershell \"\$xml = Resolve-Path ${original};\$output = Join-Path (\$pwd) temp.xml;\$xslt = New-Object System.Xml.Xsl.XslCompiledTransform;\$xslt.Load(\\\"${transform}\\\");\$xslt.Transform(\$xml, \$output);\"
```

Comment:
Hm... I wasn't able to find how, but I'm not a groovy expert, so maybe I wasn't looking at the right place :)

@emanuelez is there an easy way to apply an xslt transform in groovy?